lib/pipe/gitlab.js

```diff
@@ -19,7 +24,6 @@ class GitLabPipe {
     // GitLab PAT looks like glpat-nKGdja3jsG4850sGksh7
     this.token = params.GITLAB_PAT || this.ENV.GITLAB_PAT;
     this.hiddenCommentData = `<!--- testomat.io report ${process.env.CI_JOB_NAME || ''} -->`;
-    this.axios = axios.create();
```

Comment:
import above should be changed to `const { default: axios } = require('axios');`